README.md

```diff
@@ -59,10 +59,11 @@ functions-framework==3.*
 Create an `main.py` file with the following contents:
 
 ```python
+import flask
 import functions_framework
 
 @functions_framework.http
-def hello(request):
+def hello(request: flask.Request) -> flask.typing.ResponseReturnValue:
     return "Hello world!"
 ```
```

Comment:
I'm not sure how to best show this to users:

* On one hand, I think it would be great if developers don't have to search for a long time the type definitions of the functions
* On the other hand, I _really_ like these short examples in the README file

:thinking:

Reply:
Including these type annotations for the sake of illustration in this developer focused README.md seems like a reasonable doc update, some of the essence of the simplicity of this sample is preserved on the GCP docs in https://cloud.google.com/functions/docs/samples/functions-helloworld-get#functions_helloworld_get-python.